float8_experimental/float8_ops.py

```diff
@@ -63,6 +63,26 @@ def make_float8(data):
     return list(out)
 
 
+@implements([aten.copy_.default])
```

Comment:
I think this is superseded by the final version in my PR that already got merged

Reply:
yup will rebase